packages/wallet/src/redux/protocols/indirect-funding/reducer.ts

```diff
@@ -4,11 +4,12 @@ import * as selectors from '../../selectors';
 import * as helpers from '../reducer-helpers';
 import { getLastCommitment, ChannelState } from '../../channel-store/channel-state';
 import { CommitmentType } from 'fmg-core';
+import { isExistingLedgerFundingAction } from '../existing-ledger-funding';
+// TODO: Why does importing the reducer from the index result in test failures in grand parent protocols?
```

Comment:
Not sure why but importing from the index file caused this error ` TypeError: existing_ledger_funding_1.existingLedgerFundingReducer is not a function`  in a couple tests.

Reply:
I actually got `TypeError: existing_ledger_funding_2.initializeExistingLedgerFunding is not a function` (note the `_2`, not `_1`).

Strange.